db/seeds.rb

```
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+user = User.create({ email: 'a@a.com', password: 'asdasdasd' })
```

Use `create!`  on seeds so it fails loudly if it tries to create something invalid.